app/javascript/components/common/SearchableList.tsx

```diff
@@ -73,6 +80,7 @@ export const SearchableList = ({
           value={request.query.criteria || ''}
           placeholder={placeholder}
         />
+        {isFetching ? <span>Fetching</span> : null}
```

Comment:
@iHiD This is where we can put the spinner. I just stubbed it in for now.

Reply:
OK. Thanks.